test/release/examples/benchmarks/lcals/RunCRawLoops.chpl

```diff
@@ -63,7 +63,7 @@ module RunCRawLoops {
             for isamp in 0..#num_samples {
               q = 0.0;
               for k in 0..#len {
-                q = z[k]*x[k];
+                q += z[k] * x[k];
```

Comment:
I am not super familiar with the LCALS code. Is there some kind of check that validates the result that should have caught this? If so, is it just coincidental that this didn't impact the result?

Reply:
> It does a fake "checksum" to verify results, but this slipped by while still passing that.

Gotcha. Looks good to me then.